config/router.js

```diff
@@ -71,58 +73,74 @@ export default [
               },
             ],
           },
+          //admin capstone
           {
             path: '/admin',
             icon: 'book',
-            name: 'admin-capstone',
+            name: 'Admin',
```

Comment:
dùng name trong file menu
```suggestion
            name: 'Admin',
```